Document interface subscription capability [DPP-1221]. (#14982)
digital-asset/daml: docs/source/app-dev/services.rst

You can get these included in requests related to Transactions by setting the ``verbose`` field in message ``GetTransactionsRequest`` or ``GetActiveContractsRequest`` to ``true``.

.. _transaction-filter:

Transaction Filter
------------------

> ``TransactionService`` offers subscriptions by filtering transactions by templates or interfaces using ``GetTransactions`` by providing :ref:`transaction filter <com.daml.ledger.api.v1.TransactionFilter>` in ``GetTransactionsRequest``. Transaction Filter allows:

Reviewer: ```suggestion
``TransactionService`` offers transaction subscriptions filtered by templates and interfaces using ``GetTransactions`` calls. A :ref:`transaction filter <com.daml.ledger.api.v1.TransactionFilter>` in ``GetTransactionsRequest``. allows:
```

> - filter by a party, :ref:`inclusive <com.daml.ledger.api.v1.Filters.inclusive>` should be empty
> - filter by a party and a :ref:`template ID <com.daml.ledger.api.v1.InclusiveFilters.template_ids>`
> - filter by a party and an :ref:`interface ID <com.daml.ledger.api.v1.InterfaceFilter.interface_id>`
> - exposing an interface view by setting :ref:`include_interface_view <com.daml.ledger.api.v1.InterfaceFilter.include_interface_view>` to ``true``

Reviewer: ```suggestion
- filtering by a party, when the :ref:`inclusive <com.daml.ledger.api.v1.Filters.inclusive>` field is left empty
- filtering by a party and a :ref:`template ID <com.daml.ledger.api.v1.InclusiveFilters.template_ids>`
- filtering by a party and an :ref:`interface ID <com.daml.ledger.api.v1.InterfaceFilter.interface_id>`
- exposing an interface view, when the :ref:`include_interface_view <com.daml.ledger.api.v1.InterfaceFilter.include_interface_view>` is set to ``true``
```
